sql/src/main/java/org/apache/druid/sql/calcite/planner/PlannerFactory.java

```diff
@@ -52,7 +52,7 @@
 import java.util.Map;
 import java.util.Properties;
 
-public class PlannerFactory
+public class PlannerFactory implements PlannerToolbox
```

Comment:
why does PlannerFactory has to implement PlannerToolbox? Instead, it could take plannerToolbox as an argument. is the implementation any different from SimplePlannerToolbox?